src/modules/jcms_rest/src/EventSubscriber/ExceptionElifeJsonSubscriber.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\jcms_rest\EventSubscriber;
+
+use Drupal\Core\EventSubscriber\ExceptionJsonSubscriber;
+
+/**
+ * Handle eLife JSON exceptions the same as JSON exceptions.
+ */
+class ExceptionElifeJsonSubscriber extends ExceptionJsonSubscriber {
```

Comment:
This basically tells Drupal that if there's an exception when using the custom JSON format then handle it in the same way as the normal JSON exceptions are handled. As for the second part of the question, I'm unsure. They'd be reflected in the response via a response code but I'll need to look further into this.